docs/guides/basic-database.rst

```diff
@@ -1,194 +1,106 @@
 Basic setup: configuring database connectivity
 ==============================================
 
-This shows you how to configure the Generic MySQL backend. This backend
-is called 'gmysql', and needs to be configured in ``pdns.conf``. Add the
-following lines, adjusted for your local setup (specifically, you may
-not want to use the 'root' user):
+This shows you how to configure the Generic SQLite3 backend.
 
-.. code-block:: ini
-
-    launch=gmysql
-    gmysql-host=127.0.0.1
-    gmysql-user=root
-    gmysql-dbname=pdns
-    gmysql-password=mysecretpassword
+Make sure the SQLite3 backend is installed:
 
-Remove any earlier :ref:`setting-launch` statements and
-other configuration statements for backends.
+.. code-block:: shell
 
-.. warning::
-  Make sure that you can actually resolve the hostname of
-  your database without accessing the database! It is advised to supply an
-  IP address here to prevent chicken/egg problems!
+    $ sudo apt-get install pdns-backend-sqlite3
 
-Now start PowerDNS in the foreground:
+or
 
-::
+.. code-block:: shell
 
-    # /usr/sbin/pdns_server --daemon=no --guardian=no --loglevel=9
-    (...)
-    Dec 30 13:40:09 About to create 3 backend threads for UDP
-    Dec 30 13:40:09 gmysql Connection failed: Unable to connect to database: Access denied for user 'hubert'@'localhost' to database 'pdns-non-existant'
-    Dec 30 13:40:09 Caught an exception instantiating a backend: Unable to launch gmysql connection: Unable to connect to database: Access denied for user 'hubert'@'localhost' to database 'pdns-non-existant'
-    Dec 30 13:40:09 Cleaning up
-    Dec 30 13:40:10 Done launching threads, ready to distribute questions
+    $ sudo yum install pdns-backend-sqlite
 
-This is as to be expected - we did not yet add anything to MySQL for
-PowerDNS to read from. At this point you may also see other errors which
-indicate that PowerDNS either could not find your MySQL server or was
-unable to connect to it. Fix these before proceeding.
+This backend is called 'gsqlite3', and needs to be configured in ``pdns.conf``.
+Add the following lines, adjusted for your local setup:
 
-General MySQL knowledge is assumed in this chapter, please do not
-interpret these commands as DBA advice!
+.. code-block:: ini
 
-Example: configuring MySQL
---------------------------
+    launch=gsqlite3
+    gsqlite3-database=/var/lib/powerdns/pdns.sqlite3
 
```

Comment:
Perhaps a `.. note::` here describing the access permissions of /etc wrt to systemd?

Reply:
Why?